### DISCUSSION TITLE
RC522  RFID Reader

### DISCUSSION THREAD

**Discussion by kaper25** [arendst/Tasmota#12345] [Q&A] [answered]
Using the common RFID-RC522 reader (https://imgaz2.staticbg.com/thumb/large/oaupload/ser1/banggood/images/02/23/e5243110-fbe1-421e-abb8-1efa56c2283b.jpg.webp) works fine, but i would like to use "MINI RC522" (https://imgaz2.staticbg.com/thumb/large/oaupload/ser1/banggood/images/30/FB/fc995cc5-2004-40b5-bc2b-e3351c856e64.jpg.webp) wich should be very similar to the bigger one. The pinout is identical and board is dected by tasmota as RC522 v2.0 - but no tags can be read.  Any Idears? 

thanks in advance
erich

**Comment by barbudor** [comment, marked as answer]
From NXP MFRC522 datasheet v2.0 is supposed to be fully compatible with 1.0 and with superior performance.
The Tasmota driver doesn't make any difference except for the display of the version in the Init function.

Have you presented the tag to the antenna side that is on the other side of the components (the board may have a internal shield that prevent reading from the wrong side)
